theme/models.py

```diff
@@ -594,7 +594,26 @@ class UserProfile(models.Model):
 
     email_opt_out = models.BooleanField(default=False)
 
-    _bucket_name = models.CharField(max_length=63, null=True)
+    _bucket_name = models.CharField(max_length=63, null=True, blank=True, editable=False, default=None)
```

Comment:
I think it'd be more straight forward to just generate the _bucket_name at creation and make it immutable along with a migration that generates the _bucket_name for each existing user that doesn't have it set yet.

Reply:
good call thanks